pysatMissions/methods/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+pysatMissions.methods is a module that provides
+the methods to interface with numerous empirical model packages
+"""
+
+from pysatMissions.methods import aacgmv2
+from pysatMissions.methods import apexpy
+from pysatMissions.methods import pyglow
+from pysatMissions.methods import spacecraft
```

Comment:
Should you really be importing external modules from a local specification?

Reply:
These are internal modules, but the naming convention makes them confusing here.

Reply:
We should clarify this point of confusion:
`pysatMissions.methods.aacgmv2` is a set of routines that allow interface to `aacgmv2`.  This is based on the instrument methods convention of `pysat.instruments.methods.cdaweb`, etc.  Is there a less confusing way we should be doing this?

Reply:
Renaming the local module to not be the same name as the imported module (as I think you said you were going to do in a follow up message) sounds like a good way to go.